### PR TITLE
Fix: Help contributors know where they can find files to update when contributing to the learning resource

### PR DESCRIPTION
**Pull request by ManasviGoyal**
**This pull request makes the following changes:**
* Fixes issue #17 

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-documentation

### CODE REVIEW

**Comment by ManasviGoyal** [comment]
@Ifycode please check

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 1]
```diff
@@ -21,3 +21,7 @@ Everyone is welcome to help improve the content of the Git and Github training r
 {% hint style="info" %}
 Reload any one page from this learning resource (whenever you come back to continue this training) so that newly added content can show up (if any new content has been added).
 {% endhint %}
+
+{% hint style="info" %}
+The contributors interested in improving the contents of the learning resource can find the files to update in the [doc folder](https://github.com/Ifycode/git-github-training-resource/tree/develop/doc) of the [learning resource repo](https://github.com/Ifycode/git-github-training-resource).
+{% endhint %}
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:27]: @ManasviGoyal I love the way you were able to get how to make it a hint. Good job on that. But you made one mistake - You added the hint in the wrong place. The instruction was to add the hint just after the last list item under the **Learning resource repo: How to submit your changes**. Help change the position of the hint you added accordingly.

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by Ifycode** [comment]
Also @ManasviGoyal, let's improve your pull request title to reflect the issue you worked on so that it is easy to find. Change your pull request title from **added hint for contributors** to **Fix: Help contributors know where they can find files to update when contributing to the learning resource**

File: doc/contributors-guide/making-and-submitting-changes.md (hunk 1 above)
Comment (ManasviGoyal) [reply doc/contributors-guide/making-and-submitting-changes.md]: Thank you. I have fixed it now.

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Everyone is welcome to help improve the content of the Git and Github training r
 * Send a pull request pointing to the develop branch of the original learning resource repo.
 * Wait for review of your changes from the repo admin(s).
 
+{% hint style="info" %}
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:13]: Very well done @ManasviGoyal 
And the PR title too.
Changes approved!

**Review by Ifycode — APPROVED** [review]

**Comment by Ifycode** [comment]
@all-contributors add @ManasviGoyal for docs

File: doc/contributors-guide/making-and-submitting-changes.md (hunk 2 above)
Comment (ManasviGoyal) [reply doc/contributors-guide/making-and-submitting-changes.md:13]: Thank you.

**Pull request merged**